DotnetViewComponents/Views/Shared/Components/Image/Default.cshtml

```diff
@@ -0,0 +1,9 @@
+@using DotnetViewComponents.ViewModels
+@model ImageViewModel
+
+<figure class="nhsuk-image">
+    <img class="nhsuk-image__img" src="@Model.Src" alt="@Model.Alt" sizes="@Model.Sizes" srcset="@Model.SrcSet" @(Model.AdditionalAttributes != null ? Model.AdditionalAttributes : "")>
```

Comment:
I've never seen `srcset` used like this - https://www.w3schools.com/TAGs/att_source_srcset.asp